.buildkite/pipeline.yml

```diff
@@ -1,13 +1,2 @@
 # The Buildkite pipeline definition.
 # See https://buildkite.com/docs/pipelines for documentation.
```

Comment:
I do not know what else I need to do to disable Buildkite (and if it is okay to delete this yml file yet).